packages/angular/projects/ui-angular/src/lib/components/authenticator/components/amplify-confirm-sign-up/amplify-confirm-sign-up.component.ts

```diff
@@ -47,7 +47,7 @@ export class AmplifyConfirmSignUpComponent implements OnInit, OnDestroy {
   }
 
   onStateUpdate(state: AuthMachineState): void {
-    const actorState: SignUpState = getActorState(state);
+    const actorState = getActorState(state) as SignUpState;
```

Comment:
These assignments were getting errors only in Angular. I think there's an opportunity for types to get cleaned up to prevent `as` casting:

![Screen Shot 2021-11-04 at 9 59 04 AM](https://user-images.githubusercontent.com/15182/140353412-caa40a78-7b5d-4a68-811f-8e597f5f973a.png)
![Screen Shot 2021-11-04 at 9 59 19 AM](https://user-images.githubusercontent.com/15182/140353424-dfb64239-15b3-4415-acd7-2d79cc2b3cef.png)